packages/nextjs/hooks/scaffold-eth/useSuggestedProjects.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+
+export function useSuggestedProjects(category: any, currentProjectId: string) {
+  const [suggestedProjects, setSuggestedProjects] = useState([]);
+
+  useEffect(() => {
+    const getSuggestedProjectsByCategory = async () => {
+      try {
+        const response = await fetch(
+          `/api/suggestedProjects?category=${category}&currentProjectId=${currentProjectId}`,
```

Comment:
I think that is better than the suggested projects endpoint only receives the projectId and then the backend chooses how to select the suggested projects.